src/applications/mhv-medical-records/tests/containers/ConditionDetails.unit.spec.jsx

```diff
@@ -42,7 +42,7 @@ describe('Condition details container', () => {
     const conditionName = screen.getByText(
       initialState.mr.conditions.conditionDetails.name.split(' (')[0],
       {
-        exact: true,
+        exact: false,
         selector: 'h1',
       },
     );
```

Comment:
Rather than setting exact to false here, it might be better to remove the `split` from here as well.

Reply:
Corrected the issue.